src/tutorials/using-a-pathogen-repo/running-an-ingest-workflow.rst

```diff
@@ -0,0 +1,249 @@
+
+Download the Zika repository
+============================
+
+All pathogen ingest workflows are stored in :term:`pathogen repositories<pathogen repository>` (version-controlled folders) to track changes over time.
+Download the `Zika repository <https://github.com/nextstrain/zika>`_.
+
+.. code-block::
+
+    $ git clone https://github.com/nextstrain/zika
+    Cloning into 'zika'...
+    [...more output...]
+
+When it's done, you'll have a new directory called ``zika/``.
+
+Run the default workflow
+========================
+
+The zika :term:`ingest workflow` uses the `NCBI Datasets CLI tools <https://www.ncbi.nlm.nih.gov/datasets/docs/v2/reference-docs/command-line/>`_
+to download public data and uses a combination of :doc:`augur curate <augur:usage/cli/curate/index>` and other data manipulation tools to curate
+the downloaded data into a format suitable for :term:`phylogenetic workflows <phylogenetic workflow>`.
+
+1. Change directory to the Zika pathogen repository downloaded in the previous step
+
+.. code-block::
+
+    $ cd zika
+
+2. Run the default ingest workflow with the :term:`Nextstrain CLI`.
+
+.. code-block::
+
+    $ nextstrain build ingest
+    Using profile profiles/default and workflow specific profile profiles/default for setting default command line arguments.
+    Building DAG of jobs…
+    [...a lot of output...]
+
+This should take just a few minutes to complete.
+There should be two final output files:
+
+* ``ingest/results/metadata.tsv``
+* ``ingest/results/sequences.fasta``
+
+The output files should have the same data formats as the public data files hosted by Nextstrain, available at:
+
+* https://data.nextstrain.org/files/workflows/zika/metadata.tsv.zst
+* https://data.nextstrain.org/files/workflows/zika/sequences.fasta.zst
+
+Your results may have additional records depending on whether new data has been released since the public data files were last uploaded.
+
+
+Configuring the ingest workflow
+===============================
+
+Now that you've seen the default outputs of the ingest workflow, you can try configuring the ingest workflow to change the outputs.
+If you want to see the uncurated NCBI Datasets data to decide what changes you would like to make to the workflow, you can run the following:
+
+.. include:: ../../snippets/uncurated-ncbi-dataset.rst
+
+We'll walk through an example custom config to include an additional column in the curated output.
+For example, examining the raw NCBI metadata shows us that ``virus-name`` is a NCBI Datasets field that is not currently downloaded by the default Zika ingest workflow.
+If you wanted this field to be included in your outputs, you could perform the following steps.
+
+1. Create a new build config directory ``ingest/build-configs/tutorial/``
+
+.. code-block::
+
+    $ mkdir ingest/build-configs/tutorial
+
+2. Create a new config file ``ingest/build-configs/tutorial/config.yaml``
```

Comment:
Seeing this big YAML block made me think "whoa, were did all these values come from"? There are a couple of changes you may want to consider here:

* I'd move the text below the YAML to above it. So instead of starting with a big config file¹, start with "the parts of the config we want to change are `ncbi_datasets_fields` (because...), `field_map` (because...) and `metadata_column` (because). 
* Then detail that we copy certain sections from the existing config then add the changes just described.

¹ The config YAML alone takes up the full height of a laptop screen

Reply:
Updated in https://github.com/nextstrain/docs.nextstrain.org/pull/195/commits/9255b1a21d1564c77fe51be7e31928a2ee78f7f4